documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,46 @@
+:orphan:
+
+Release Notes
+=============
+
+We are using `towncrier <https://pypi.org/project/towncrier/>`_ to handle
+our relase notes, and this directory  contains the input for it -
+"news fragments" which are short files that contain a small
+**ReST**-formatted text that will be added to the next version's
+Release Notes page.
+
+Make sure to use full sentences with correct case and punctuation,
+and use Sphinx references
+(see https://sphinx-tutorial.readthedocs.io/cheatsheet/).
```

Comment:
Do we want:
```
TheFeature
----------
Summary of new feature
```
or

```
* Summary of new feature
```
or 
```
Summary of new feature
```

as the format in each fragment? We should give an example of whatever we pick in this file (I don't care which)

Reply:
Added (the last one since the bullet point is added by towncrier in our configuration).